Test/Unit/Plugin/Customer/Model/CustomerDataTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Klaviyo\Reclaim\Test\Unit\Plugin\Customer\Model;
+
+use PHPUnit\Framework\TestCase;
+use Magento\Customer\CustomerData\Customer;
+use Magento\Customer\Helper\Session\CurrentCustomer;
+use Magento\Customer\Api\Data\CustomerInterface;
+use Klaviyo\Reclaim\Plugin\Customer\Model\CustomerData;
+
+class CustomerDataTest extends TestCase
+{
+    /**
+     * @var Customer
+     */
+    protected $object;
+
+    protected function setUp()
+    {
+        $customerInterfaceMock = $this->createMock(CustomerInterface::class);
+        $customerInterfaceMock->method('getLastname')->willReturn('Smith');
```

Comment:
are you able to access the constants from the other file for these values? otherwise if they get updated the tests will fail.